README.md

```diff
@@ -76,6 +76,13 @@ Example:
 $ REACT_APP_BUILD_MODE=single-cluster yarn build
 ```
 
+## Container image build
+
+You can build the container image by running:
```

Comment:
@mhrivnak IMHO people my also benefit of an example showing how to override the ASSISTED_SERVICE_URL